content/02.introduction.md

```diff
@@ -67,14 +72,30 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
+In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10/ggnxb3] described the clinical features of patients infected with SARS-CoV-2 including lower respiratory tract infection with fever, dry cough, and dyspnea. 
+[@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common which suggests that the virus targets cells located in the lower respiratory tract. 
+Recent reports suggest that symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation. 
+[@doi:10/ggnxb3] also found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
 Mortality also appeared to be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
 They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
 
-
 #### Vaccines for Viruses: Strategies for and challenges to development
 
-What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?
+Generally, the first step in producing a vaccine is characterizing the target.
+The genetic sequence of SARS-CoV-2 was released to the world on January 11, 2020, which launched an intense global effort to develop a vaccine to prevent COVID-19.
+The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and vaccine developers to develop vaccines against SARS-CoV-2.
+As of April 8, 2020, there are 115 vaccine candidates to prevent COVID-19, of which 78 were active. 
+Of the 78 active vaccine programs, 73 were categorized as in the preclinical or exploratory stage (@doi:10.1038/d41573-020-00073-5).
+
+Historically, an H1N1 influenza vaccine was developed relatively efficiently, mainly because influenza-vaccine technology had already been developed and regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
+Critiques of the experience producing and distributing the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
+Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into the commercially available seasonal influenza vaccines (@doi:10.1056/NEJMp2005630).
```

Comment:
I think this part needs to be more condensed, especially since it is the intro for vaccines for COVID-19 and this feels a little bit off topic for the intro

Reply:
There note in this section said, "What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?" I wouldn't have thought to necessarily talk about H1N1 otherwise.